apps/client/src/app/auth/_state/server/api.ts

```diff
@@ -1,10 +1,9 @@
-import type { SignInFormValues } from "@auth/_types";
+import type { SignInFormValues, SignUpFormValues } from "@auth/_types";
 import axios from "axios";
+import { AxiosInstance } from "~/app/shared/server";
 
-export const signUpApi = {
-  signUpResponse: (postData: object) => {
-    return axios.post("/api/signup", { postData });
-  },
+export const authApi = {
+  signUp: (formData: SignUpFormValues) => AxiosInstance({ isRequireAuth: false }).post("api/v1/auth/join", formData),
 };
 
 export const LoginApi = {
```

Comment:
카멜 케이스와 파스칼 케이스 두 가지가 보입니다!
카멜 케이스로 통일하는게 좋아보입니다 :)
```suggestion
authApi / loginApi
```

Reply:
아 제 실수입니다! 수정해놓겠습니다.
본 작업은 sign-up 브랜치 pr 완료 후, login 브랜치에서 정정해놓도록 하겠습니다. 😄